dependencies.yaml

```diff
@@ -0,0 +1,101 @@
+# Dependency list for https://github.com/rapidsai/dependency-file-generator
+files:
+  all:
+    output: conda
+    matrix:
+      cuda: ["11.5"]
+      arch: [x86_64]
+    includes:
+      - build
+      - cudatoolkit
+      - develop
+      - doc
+      - run
+      - test_python
+  test_python:
+    output: none
+    includes:
+      - cudatoolkit
+      - py_version
+      - test_python
+  test_cpp:
+    output: none
+    includes:
+      - cudatoolkit
+      - test_cpp 
+  checks:
+    output: none
+    includes:
+      - build
+      - develop
+      - py_version
+channels:
+  - rapidsai
+  - rapidsai-nightly
+  - conda-forge
+  - nvidia
+dependencies:
+  build:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - cudf=22.12.*
+          - cmake>=3.23.1
+          - cython>=0.29,<0.30
+          - python>=3.8,<3.10
+          - scikit-build>=0.13.1
+          - gdal>=3.0.2
+          - geopandas>=0.11.0
+  cudatoolkit:
+    specific:
+      - output_types: conda
+        matrices:
+          - matrix:
+              cuda: "11.5"
+            packages:
+              - cudatoolkit=11.5
+  develop:
+    common:
+      - output_types: [conda, requirements]
+        packages:
+          - pre-commit 
+      - output_types: conda
```

Comment:
These clang packages can also be removed after #799, which uses pre-commit to handle clang-format.

Reply:
Are these used for `clang-format` or used to setup language servers for intellisense?

Reply:
I think these are just for `clang-format`. The language servers are independently managed (in rapids-compose, at least), and I think this clang version is too old for that anyway.